src/Facet/Elab.hs

```diff
@@ -291,50 +296,43 @@ data StaticContext = StaticContext
 
 data ElabContext = ElabContext
   { context :: Context
-  , sig     :: [Interface Type]
+  , sig     :: [Signature Type]
```

Comment:
Making this a list of signatures instead of a list of interfaces mirrors the structure of the context, in that a single argument can potentially bring multiple interfaces into scope at once. Indexing will thus be able to reflect the binding structure of the language directly in the same manner as contexts.